cmake/serac-config.cmake.in

```diff
@@ -94,17 +94,18 @@ if(NOT SERAC_FOUND)
 
   # Caliper
   if(SERAC_USE_CALIPER)
-    # See comment in SetupSeracThirdParty.cmake
-    if(SERAC_ENABLE_CUDA)
-      if(CMAKE_VERSION VERSION_LESS 3.17)
-        message(FATAL_ERROR "Serac+Caliper+CUDA requires CMake > 3.17.")
-      else()
-        find_package(CUDAToolkit REQUIRED)
-      endif() 
-    endif()
     find_dependency(caliper REQUIRED PATHS "${CALIPER_DIR}" "${CALIPER_DIR}/share/cmake/caliper")
   endif()
 
+  # See comment in SetupSeracThirdParty.cmake
+  if(SERAC_ENABLE_CUDA)
+    if(CMAKE_VERSION VERSION_LESS 3.17)
+      message(FATAL_ERROR "Serac+CUDA requires CMake > 3.17.")
+    else()
+      find_package(CUDAToolkit REQUIRED)
+    endif() 
+  endif()
```

Comment:
`CUDAToolkit` is now always required with CUDA, to find `cublasLt`

https://github.com/LLNL/serac/blob/develop/cmake/thirdparty/SetupSeracThirdParty.cmake#L44-L47